### DISCUSSION TITLE
钱没退给我，帐号直接变免费版了，太离谱

### DISCUSSION THREAD

**Discussion by seagull2024** [getlantern/lantern-forum-cn#1056] [General]
之前多交了一年的费用,要交退款，钱没退，现在帐号直接变成免费版了，实在太离谱
之前退款申请链接：[https://github.com/getlantern/lantern-forum-cn/discussions/1033](url)
https://github.com/getlantern/lantern-forum-cn/discussions/1033

**Comment by Ginny125** [comment]
我也这样，在支付宝账单里申请退款了，现在等结果中

**Reply by seagull2024** [reply]
我找不到退款入口，在支付宝哪个页面？

**Reply by Ginny125** [reply]
历史账单里找到你当时付款的订单，翻到最下面，对此订单有疑问，点进去就可以申诉了，但是没啥用，我这里显示商家已处理，但还是没退款，你可以试试

**Comment by MaxMKMax** [comment]
@seagull2024 您好. 那是系统问题. 请您提供我支付单据和账号邮箱. 我让技术团队帮您恢复专业版.

**Reply by seagull2024** [reply]
我不要恢复专业版，我之前链接你没看么，帮我退钱就可以了。

**Reply by MaxMKMax** [reply]
@seagull2024 您好. 好的明白. 我再帮您提交退款. 那个链接不知道为什么 404 了. 请您重新提供信息. 谢谢.

**Reply by Ginny125** [reply]
@Mike777Khoo 也看看我的帖子啊，支付截图和账号都写的明明白白，就回复一句然后冷处理？上周蓝灯就变免费版了，一直不退钱

**Reply by seagull2024** [reply]
> @seagull2024 您好. 好的明白. 我再帮您提交退款. 那个链接不知道为什么 404 了. 请您重新提供信息. 谢谢.

https://github.com/getlantern/lantern-forum-cn/discussions/1033

**Reply by seagull2024** [reply]
> @seagull2024 您好. 好的明白. 我再帮您提交退款. 那个链接不知道为什么 404 了. 请您重新提供信息. 谢谢.

[要求退款链接](https://github.com/getlantern/lantern-forum-cn/discussions/1033)

**Comment by seagull2024** [comment]
[链接](https://github.com/getlantern/lantern-forum-cn/discussions/1033)

**Comment by seagull2024** [comment]
这么久了，一直没处理。拖字决啊
